sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/tests/Azure.Monitor.OpenTelemetry.LiveMetrics.Demo/Program.cs

```diff
@@ -17,6 +17,9 @@ internal class Program
 
         private static Random _random = new();
 
+        private const int chunkSizeMB = 1000;
```

Comment:
Do you really want to generate such a big allocation?

Reply:
I needed a large allocation to trigger GC and see ups and down on the chart. 
I'll discuss with you offline.

Reply:
Let me merge this for now and i'll revisit. I also want this demo to show CPU usage.

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/42781